docs/.vuepress/public/_redirects

```diff
@@ -3,6 +3,9 @@
 # LOTUS SPECIFIC REDIRECTS
 # ------------------------
 /get-started/store-and-retrieve/ https://lotus.filecoin.io/docs/tutorials/store-and-retrieve/
+/get-started/store-and-retrieve/set-up/ https://lotus.filecoin.io/docs/tutorials/store-and-retrieve/
+/get-started/store-and-retrieve/store-data/ https://lotus.filecoin.io/docs/tutorials/store-and-retrieve/
+/get-started/store-and-retrieve/retrieve-data/ https://lotus.filecoin.io/docs/tutorials/store-and-retrieve/
```

Comment:
These redirects aren't actually happening yet because the **Store and retrieve** tutorial is still in the repo. There's an [issue in the lotus-docs repo discussing it](https://github.com/filecoin-project/lotus-docs/issues/27).